src/main/java/uk/gov/companieshouse/mapper/chips/DissolutionChipsMapper.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.companieshouse.mapper.chips;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.model.db.dissolution.Dissolution;
+import uk.gov.companieshouse.model.dto.chips.ChipsForm;
+import uk.gov.companieshouse.model.dto.chips.ChipsFormAttachment;
+import uk.gov.companieshouse.model.dto.chips.ChipsPackageMetadata;
+import uk.gov.companieshouse.model.dto.chips.DissolutionChipsRequest;
+
+import java.util.Base64;
+import java.util.Collections;
+
+@Service
+public class DissolutionChipsMapper {
+
+    private static final int PACKAGE_METADATA_COUNT = 1;
+
+    private static final String FORM_CATEGORY = "FORM IMAGE PDF";
+    private static final String FORM_MIME_TYPE = "application/pdf";
```

Comment:
Can we put this in `src/main/java/uk/gov/companieshouse/model/Constants.java` as well?

Reply:
Sure - I'd been keen to keep the CHIPS-specific stuff inside the mapper though as they are global constants.

Reply:
done